packages/fast-development-site-react/app/app.tsx

```diff
@@ -123,10 +128,19 @@ export default class App extends React.Component<{}, AppState> {
             component: Paragraph,
             status: Status.alpha,
         };
+        const componentObj2: any[] = [{ text: "uno" }, { text: "due" }, { text: "tre" }];
+        const categoryObj2: SiteCategoryProps = {
```

Comment:
can you rename this to `spanTestCategory` and the paragraph version to `paragraphTestCategory`